configextractor.py

```diff
@@ -151,3 +170,22 @@ def classification_checker(res_section, parser_name, file_parsers):
         if name == parser_name:
             res_section.classification = cl_engine.normalize_classification(parser_obj.classification)
     return res_section
+
+
+def subsection_builder(parent_section: ResultSection = None, fields: dict = {}):
+    for field, tag in FIELD_TAG_MAP.items():
+        if field in fields:
+            generic_section = ResultSection(f"Extracted {field.capitalize()}")
+            field_data = fields[field]
+            if tag:
```

Comment:
do we want to just display the tags or display both the tags and a line. I think it's nice to have both, would be easier for someone to read quickly when scrolling through. This is how it is currently 
![image](https://user-images.githubusercontent.com/62077906/98416056-68a24380-204c-11eb-9bb0-29128396317b.png)
@cccs-jp thoughts

Reply:
@cccs-aa agreed!

Reply:
I agree it would be easier to read if someone is going through the service result, but there's also a lot of duplicated information between the tags and the section content, and there is a lot of whitespace that could be filled.
![image](https://user-images.githubusercontent.com/59843993/98546009-513da300-2264-11eb-8c47-37ed9b538fa5.png)
Maybe there is a better way of displaying this information by utilizing a table for specific parts of the config? Something like this perhaps
![image](https://user-images.githubusercontent.com/59843993/98547256-e55c3a00-2265-11eb-93b4-36121591bcf0.png)

Reply:
I looked into this a while ago, it's possible but we can't really use the current table section for this because the fields don't fit the needs for this service. Would have to create a custom table and I'm not sure it would be much more useful.

Reply:
From discussion today, it may be worth exploring a table format that contains (at least) one column with the header C2, with each row `some_ip:some_port` or `some_domain:some_port`. This way the section will have a fixed height and a vertical scrollbar will appear if the number of values causes the section height to grow too large.